packages/adapter-postgresjs/test/index.test.ts

```diff
@@ -0,0 +1,41 @@
+import { runBasicTests } from "utils/adapter"
+import PostgresJSAdapter from "../src"
+import postgres from "postgres"
+
+const sql = postgres({
+  host: "127.0.0.1",
+  database: "pgjs",
+  user: "pgjs",
+  password: "pgjs",
+  port: 5432,
+  max: 20,
+})
+
+runBasicTests({
+  adapter: PostgresJSAdapter(sql),
+  db: {
```

Comment:
It looks like you've added the `Authenticator` (PassKeys) methods, but didn't add the tests.

Can you add the `testWebAuthnMethods: true` option here and the `db: authenticator() ..` method? Check out the Prisma `index.test.ts` as an example (`packages/adapter-prisma/test/index.test.ts`)

Reply:
> It looks like you've added the `Authenticator` (PassKeys) methods, but didn't add the tests.
> 
> Can you add the `testWebAuthnMethods: true` option here and the `db: authenticator() ..` method? Check out the Prisma [index.test.ts](https://github.com/nextauthjs/next-auth/blob/main/packages/adapter-prisma/test/index.test.ts?rgh-link-date=2024-07-21T09%3A50%3A28Z#L11) as an example

Alright.. ive added it!